pkgs/development/tools/cocogitto/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+rustPlatform.buildRustPackage rec {
+  pname = "cocogitto";
+  version = "4.0.1";
+  src = fetchFromGitHub {
+    owner = "oknozor";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-uSKzHo1lEBiXsi1rOKvfD2zVlkAUVZ5k0y8iiTXYE2A=";
+  };
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  doCheck = false;
```

Comment:
Can you add a comment why the tests can't be run?

Reply:
some test fall, I haven't looked in detail why.  just assumed it was due to nix and not the  package. considering the tool's test work as intended outside of build env

Reply:
so I have look more closely at the  error is seem to depend on some git setting being already set in to home environment of the build system,  Ill be updating this later today. sorry for the wait

Reply:
If you feel like it, it's very common for derivations to setup git a bit in preCheck, see [here](https://sourcegraph.com/search?q=context:global+git+config+repo:nixos/nixpkgs&patternType=literal) for examples.

Reply:
cool ill see what i can do to fix it up a bit